client/public/locales/es/translation.json

```diff
@@ -15,9 +15,9 @@
   "Typewriter mode": "Modo línea a línea",
   "information, Impressum, privacy policy": "información, Impressum, política de privacidad",
   "Preferences": "Preferencias",
-  "Game Info & Credits": "Información del juego y reconocimientos",
+  "Game Info & Credits": "Información del juego y créditos",
```

Comment:
"Reconocimientos" is closer to acknowledgements, whereas "créditos" is a more direct translation.